assets/js/main.js

```diff
@@ -3,6 +3,36 @@ $(function(){
     $('.sidenav, .overlay').toggleClass('is-visible');
     e.preventDefault();
   });
+
+  var generateCodeSnippets = function(content, previewBox) {
+
+    var self = this;
+
+    this.parseCode = function(previewBox) {
+      var sampleCode = $('<div>');
+      $(sampleCode).html($(previewBox).html())
+      $(sampleCode).find('.is-peripheral').remove();
+      return sampleCode;
+    }
+
+    this.render = function(previewBox, sampleCode) {
+
+      var sampleCodeBox = $('<div class="usa-accordion-bordered usa-code-sample"><ul class="usa-unstyled-list"><li><button class="usa-button-unstyled" aria-expanded="false" aria-controls="collapsible-0">Code</button><div id="collapsible-0" aria-hidden="true" class="usa-accordion-content"><pre><code class="language-markup"></code></pre></div></li></ul></div>');
```

Comment:
I think my comment was lost, but could you clean up this huge string by breaking it up into lines and tabs?